cirq-google/cirq_google/engine/calibration_to_noise_properties.py

```diff
@@ -156,21 +161,40 @@ def noise_properties_from_calibration(
 
     # 5. Extract entangling angle errors.
     fsim_errors = {}
+    gate_zphase_code_pairs: Dict[Type['cirq.Gate'], str] = {
```

Comment:
Should this be a CONSTANT and put at the top?

Reply:
Done for this and similar constants in the function. (Also made `gate_times_ns` an argument to partially address the dangling TODO)